docs/node-operators/get-started/setup-validator.md

```diff
@@ -302,37 +302,27 @@ Once you have tokens, connect your Ethereum wallet and your Vega Wallet, and ass
 ```
 
+Setting the optional argument `--submitter-address` triggers the Vega network to automatically issue signature bundles that can be used to update signer set changes on the Multisig Control contract. This means if your node is promoted to a consensus validator it is easier for you to add your node's Ethereum key to the contract and to continue receiving rewards. See [maintaining the multisig contract ↗](../how-to/maintain-multisig-contract.md) for more information.
+
+
 ## Nominate your node
 To move on to self-staking, wait until you see your node on the validator list by querying the API or checking the [governance dApp ↗](https://validator-testnet.governance.vega.xyz/staking/).
 
 Then, associate your tokens and nominate your node using the [governance dApp ↗](https://validator-testnet.governance.vega.xyz/staking/) or by interacting directly with the smart contract.
 
-## Submit signature bundle
-In the epoch after you announced your node, and once your pending validator has seen transactions and sent heartbeats, your node will be added to a signature bundle built by the network: proof from the network that your node can be added to the multisigControl signers.
-
-Use the [add signer tool ↗](https://validator-testnet.tools.vega.xyz/) to submit the signature bundle. You'll need your node ID, which you can find on the latest epoch's [validator list](https://api-validators-testnet.vega.rocks/api/v2/epoch).
-
-Once the signature bundle is accepted, your node will be able to emit signatures to control withdrawals from the bridge.
-
-:::caution
-You need to add your node as a signer by the end of the epoch that the signature is emitted in. If you do not, the node will be demoted and removed from the signer set.
-:::
-
 ## Announce node off-chain
 [Create a validator profile on the forum ↗](https://community.vega.xyz/c/mainnet-validator-candidates/23) describing the experience you have, security practices and policies, how you will ensure maximum uptime, how you'll meet suitable performance standards, your communication channels for questions and the role you intend to take in Vega's governance.
 
 Share your profile with the community, for example in [the Validators Discord channel ↗](https://discord.com/channels/720571334798737489/869236034116943903), to attract stakers to nominate your node.
 
 ## Next steps
-Once your node is up and running, you'll need to maintain it, and ensure that it's still taking part in the network. You may also need to upgrade the software, restart the network and rotate your keys (for security). 
+Once your node is up and running, you'll need to maintain it, and ensure that it's still taking part in the network. If your node is promoted into the consensus validator set then the Multisig Contract will need to be updated so that you can continue to receive rewards. You may also need to upgrade the software, restart the network and rotate your keys (for security). 
```

Comment:
```suggestion
Once your node is up and running, you'll need to maintain it, and ensure that it's still taking part in the network. If your node is promoted into the consensus validator set then the Multisig Contract will need to be updated so that you and your nominators can continue to receive rewards. You may also need to upgrade the software, restart the network and rotate your keys (for security). 
```